app/ui/browser/views/page/page.jsx

```diff
@@ -27,6 +27,11 @@ const PAGE_STYLE = Style.registerStyle({
   // Mark this as the relative anchor for floating children (e.g. search bar).
   position: 'relative',
   flex: 1,
+  '&:not(.active-browser-page)': {
```

Comment:
If this is from `https://github.com/electron/electron/blob/master/docs/api/web-view-tag.md#css-styling-notes` we should comment it as such